surf/apps/themes/admin.py

```diff
@@ -45,5 +45,5 @@ class ThemeAdmin(admin.ModelAdmin):
     """
     Provides admin options and functionality for Theme model.
     """
-
     form = ThemeForm
+    raw_id_fields = ("title_translations", "description_translations",)
```

Comment:
This changes the way that people can select translations in the admin. It makes searching a lot easier and performance will be optimal even when localisation database tables become very large.